python/tvm/relay/expr.py

```diff
@@ -487,6 +487,8 @@ def const(value, dtype=None):
     - bool maps to "bool"
     - other using the same default rule as numpy.
     """
+    if dtype == 'float': # for numpy, float is float64, for us it is float32
+        dtype = 'float32'
```

Comment:
Can this be merged to L496-502?

Reply:
It didn't seem natural to me because L496-502 deal with `dtype is None`.

Reply:
So what do you suggest as a better position?